### DISCUSSION TITLE
Question 103: eparties cleanup of Smithsonian records

### DISCUSSION THREAD

**Discussion by cmaves3** [fieldmuseum/EMu-Documentation#911] [Q&A] [answered]
Description: There are many duplicate records for the Smithsonian Institute across several departmental master lists. We should work to consolidate these records. Some matches are easy to identify while some may require a bit of looking into and/or consulting across departments. See link to spreadsheet. The records have been sorted into preliminary groups but you are not restricted to merging within those groups.

See Consolidating Parties Sheet > Organisation - Smithsonian tab
https://docs.google.com/spreadsheets/u/0/d/11pFwe-9bqg7iO-BqqlV6_tSI7N7Ma6X51zptJGDBvJM

In conjunction with this, we must decide how to structure these records in the Organisation tab. SI is an entity that has many constituent parts such as the National Museum of Natural History (NMNH). The museum has departments including Paleobiology. If you structure the EMu record with SI, NMNH, and Paleo as three separate fields then “Paleobiology” ends up in the Branch field so searching for it in the Department field will not yield results. If we want the dept name to be in the dept field then we will need to uniformly concatenate SI and NMNH (Ex. Smithsonian Institution - National Museum of Natural History).

**Comment by rondlg** [comment, marked as answer]
Response: Individual collections should clean up and merge as far as possible. A followup discussion in repo then needs to be had about a shared hierarchy for Smithsonian moving forward.

See data cleanup issue #918  for further discussions and decisions.

**Comment by rbaquiran** [comment]
Insects is done.

**Reply by cmaves3** [reply]
Thanks for working on this! I see that you took care of the Insect Master records but there are also a few FMNH Masters for the SI Entomology Dept. that it looks like are just used by Insects. They are on the sheet in the same section you were working on. Take a look and see if you think they could also be condensed.

I have a couple recommendations for the record that you are merging into (228152). You have the door number and the PO Box as 2 attached Address records but I suggest entering them as the Physical and Postal address, respectively, in the Address tab. For Organisation records, it is appropriate to enter the address directly in the Address tab; for Person records we want to leave the address blank and have it auto-fill from the preferred Association. Also, I've noticed from other departments' records that SI usually has a "MRC" number for the mail room/route as the whole museum shares a PO Box number. If you know the current or primary one, you may want to add that to the Postal Address. 
You should also check out the Person records that are attached to 228152 as there are a couple different SI addresses manually typed into them. You can decide if there is a need to retain that in some way but, again, the Person records should have the Address auto-fill from Associations. If you delete the address text and click the yes checkbox in the Address tab it should auto-fill. Let us know if you have an issue with that. 
Unfortunately, the EMu downtime is about to start so this will have to wait but we will have an update at the next EMu meeting on Tuesday.

We're set up for continued discussion [here](https://github.com/fieldmuseum/EMu-Documentation/issues/918)

Thanks again, 
Chris
